tools/commit_slider_tool/commit_slider.py

```diff
@@ -0,0 +1,75 @@
+import subprocess
+from argparse import ArgumentParser
+import os
+import shutil
+import sys
+from distutils.dir_util import copy_tree
+from utils.helpers import safeClearDir, absolutizePaths
+import json
+
+parser = ArgumentParser()
+parser.add_argument("-c", "--commits", dest="commitSeq", help="commit sequence")
+parser.add_argument("-cfg", "--config", dest="configuration", help="configuration source")
+parser.add_argument("-wd", "--workdir", type=bool, dest="isWorkingDir", default=False, help="flag if current directory is working")
```

Comment:
not sure that this construction is work, as input of argparser you got string in command line which always cast to True, no matter what user specify. Probably better to use actions `store_true` which will work in following way:
if user provide flag -wd, then it will be True, otherwise False

see https://docs.python.org/3/library/argparse.html#action

Reply:
Applied